src/frontend/CartPage/CartPage.jsx

```diff
@@ -1,8 +1,57 @@
-import React from "react";
+import { AiOutlineHeart } from "react-icons/ai";
+import { IoTrashBinOutline } from "react-icons/io5";
+import { useCart } from "../Contexts/CartContext/CartContext";
 import "./CartPage.css";
 const CartPage = () => {
+  const {
+    cartState: { cartItems },
+    cartDispatch,
+  } = useCart();
   return (
-    <div className="container">
+    <>
+      <div className="cart-items-wrapper">
+        {cartItems.map((cartItem) => {
+          const { _id, img, title, price, discount, ratings, desc } = cartItem;
+
+          return (
+            <div key={_id} className="price-card">
+              <AiOutlineHeart className="wishlist-icon" />
+              <span className="price-card-img">
+                <img src={img} alt="" />
```

Comment:
We should give a proper `alt` attribute every time cause that helps with accessibility. In this case, you can give `title` as an `alt` attribute. 
```javascript
<img src={img} alt={title} />
```